packages/@aws-cdk/aws-servicecatalog/README.md

```diff
@@ -31,6 +31,7 @@ enables organizations to create and manage catalogs of products for their end us
   - [Sharing a portfolio with another AWS account](#sharing-a-portfolio-with-another-aws-account)
 - [Product](#product)
   - [Adding a product to a portfolio](#adding-a-product-to-a-portfolio)
+- [TagOptions](#tag-options)
```

Comment:
This should be under the Portfolio chapter, no?

Reply:
It was originally we decided to move it up as the following PR will add the same functionality to Product